pkg/common/common.go

```diff
@@ -75,6 +76,7 @@ const (
 // Plugin names
```

Comment:
This is confusing for me. Plugin names are aws, gcp, etc, for me. This is only used as the container image name of each plugin, right?

Reply:
The container name is the only actual "name" string that's used in the velero side of the configuration -- as the container name. aws/gcp/etc. only exist as values for the velero default plugin list. So from a velero point of view, this is the plugin name.